package-scope.qmd

```diff
@@ -1,55 +1,43 @@
-2. If depended on, {ggplot2} would potentially install it's own dependencies on a user's device.
+Plots using {ggplot2} are immensely powerful. Package functions would either need to be highly specific or  complex to manage {ggplot2}'s functionality. We consider this out of scope, if we can provide code for {ggplot2} in vignettes directly.
 
-## Alternative options
+---
 
-There is a middle ground between providing users with dependency laden plotting functions, and relying on them to write their own or copy from documentation. This is to [extend {ggplot2}](https://ggplot2.tidyverse.org/articles/extending-ggplot2.html) by providing custom `geom` objects (for example [{ggtree}](https://github.com/YuLab-SMU/ggtree)) or `scales` objects (no examples outside of {ggplot2} exist for these). At present, there are no Epiverse-TRACE packages using this strategy.
+We provide more detailed plotting examples in our vignettes (approach 2) to provide avenues to start building more customized plots. Examples include [{epichains}](https://epiverse-trace.github.io/epichains/articles/epichains.html#plotting) and [{epiparameter}](https://epiverse-trace.github.io/epiparameter/articles/epiparameter.html#plotting-epidemiological-distributions)
 
-Another option is to provide a separate R package whose sole purpose is to provide plotting functions. This gives the user the choice to install the plotting package if deemed necessary. This has been found to be a good option to enhance the maintenance of the core package and it's plotting accessory package. A marginal downside of this approach is requiring the user to install and attach both packages, as well as the assumption that they know the plotting package exists. There are currently no plotting-specific packages within Epiverse-TRACE.
+With specific code examples in vignettes we want to provide a resource from which desired plots can be produced. Vignettes use base R or the {ggplot2} package to provide easily extensible plots with the addition of [layers](https://rpubs.com/hadley/ggplot2-layers). These are more maintainable than including a dependency relation in the package directly, and allow us to provide opinionated plot versions at low cost or exposure to people using the package.
 
-## Flexible philosophy
+## Alternatives
 
-Lastly, most Epiverse-TRACE R packages are under active development and what is considered optimal and discussed above may evolve over time. User feedback and user training may elucidate a need that was overlooked in initial development and the inclusion or exclusion of dependencies may change.
+Alternatively to our approach here, one could provide custom `geom` objects (for example [{ggtree}](https://github.com/YuLab-SMU/ggtree)) or `scales` objects (no examples outside of {ggplot2} exist for these). This could balance dependencies with maintainability in some scenarios. At present, no Epiverse packages use this strategy.
```

Comment:
The {grates} package exports custom `scales`. Not sure it's worth mentioning here as an example. IIRC there is a general lack of documentation on how to build custom `scales`.